config/default.schema.json

```diff
@@ -1272,6 +1272,11 @@
 			"default": false,
 			"description": "Enables the configuration of custom parameters for context external tools"
 		},
+		"FEATURE_COLUMN_BOARD_EXTERNAL_TOOLS_ENABLED": {
```

Comment:
Should we check if ctl feature is enabled when we checking if this env is enabled?